toolkit/tools/downloader/downloader.go

```diff
@@ -75,17 +78,6 @@ func main() {
 	if *dstFile != "" && *prefixDir != "" {
 		logger.Log.Fatalf("Cannot specify both --output-file and --directory-prefix")
 	}
-	if *dstFile == "" {
-		// strip query strings from url
-		u, err := url.Parse(*srcUrl)
```

Comment:
It seems this check of url validity only happens when dstFile is not specified. It's more like a step to extract the rpmFilename. Since the urls and rpmFilename are passed separately, do we still want to call `url.Path`? @dmcilvaney @PawelWMS

Reply:
I'm fine with adding more validation of the URLs.

Reply:
I'm thinking maybe we can do the validation regardless whether `dstFile` is specified? @dmcilvaney